libcxx/test/libcxx-03/minimal_cxx11_configuration.pass.cpp

```diff
@@ -0,0 +1,130 @@
+//===----------------------------------------------------------------------===//
```

Comment:
I would argue that this test doesn't add much value anymore. If Clang doesn't support an extension and we happen to use it, our tests will fail when we try to use it. I don't see what's actionable if this test fails. I'd remove the original test instead.